src/rules/comment-no-empty/README.md

```diff
@@ -0,0 +1,40 @@
+# comment-no-empty
+
+Disallow empty comments.
+
+```css
+ a { /* */ display: block; }
```

Comment:
There's a lot of distraction in the docs and tests.  Lets simplify them down e.g.

```css
    /* */
/** ↑
 * Comments like this */
```